core/src/subject/SDNADecorators.ts

```diff
@@ -75,14 +75,20 @@ interface PropertyOptions {
     initial?: string,
     required?: boolean,
     resolve?: boolean,
+    resolveLanguage?: string;
 }
 export function subjectProperty(opts: PropertyOptions) {
     return function <T>(target: T, key: keyof T) {
         target["__properties"] = target["__properties"] || {};
         target["__properties"][key] = target["__properties"][key] || {};
         target["__properties"][key] = { ...target["__properties"][key], ...opts }
+
+        if (opts.resolveLanguage) {
+            const value = key as string
+            target[`set${capitalize(value)}`] = () => {}
+        }
+
```

Comment:
As per our discussion earlier today and my comments in https://github.com/fluxsocial/flux/pull/366, `resolveLanguage` should not be what decides if we generate a setter. Instead add an explicit option to the decorator: `writable: true/false`.